minimp3.h

```diff
@@ -889,7 +896,7 @@ static void L3_midside_stereo(float *left, int n)
         VSTORE(right + i, VSUB(vl, vr));
     }
 #endif /* HAVE_SIMD */
-    for (; i < n; i++)
+    i = (n >> 2) << 2;
```

Comment:
This change is not correct. Tests failing.

Reply:
Im go to see that.